packages/hash/frontend/src/blocks/page/createFormatPlugins/LinkModal.tsx

```diff
@@ -45,68 +78,174 @@ export const LinkModal: FunctionComponent<LinkModalProps> = ({
         />
       </form>
 
-      <div className={tw`text-gray-700`}>
+      <div style={{ color: "#374151" }}>
         {!savedLinkMarkHref && isValidLink(newLinkHref) && (
           <button
-            className={tw`bg-transparent cursor-pointer text-sm text-gray-700 hover:bg-gray-200 bg-gray-200 flex items-center w-full px-4 py-1 my-1`}
+            style={{
+              alignItems: "center",
+              backgroundColor: "#E5E7EB",
+              color: "#374151",
+              cursor: "pointer",
+              display: "flex",
+              fontSize: "0.875rem",
+              lineHeight: "1.25rem",
+              marginBottom: "0.25rem",
+              marginTop: "0.25rem",
+              paddingBottom: "0.25rem",
+              paddingLeft: "1rem",
+              paddingRight: "1rem",
+              paddingTop: "0.25rem",
+              width: "100%",
+              // :hover { background-color: #E5E7EB; }
```

Comment:
You can use these selectors if you make this a Box component instead. I'd prefer that then this PR actually breaking styles as it does now.

Reply:
I agree with using Box instead. That may require a bit more effort so I'd leave it out of this PR's scope if that's OK. Many of these boxes will be eventually removed during large refactors so by not doing further conversions now, we can actually save time. When someone works on a feature, sees broken styles and the `// :hover` comment, that can be the right time to clean things up a bit more.

Reply:
I just don't think this PR was pitched as "this will break styles" and that there's any kind of collective agreement that that's okay. Has that discussion happened and I've missed it?